### PR TITLE
Add network policy specification

### PR DESCRIPTION
**Pull request by machacekondra**
Add new `kubevirt_app_network_policies` parameter to `app_deployment` role to manage Kubernetes `NetworkPolicy` object.

### CODE REVIEW

**Comment by mmazur** [comment]
Great, we're getting the same kinds of random CI errors that were happening on Travis, on the same tasks. (I'm getting a PVC creation on the next task on my PR.) I need to start bugging relevant people, because this looks like some kind of issue with kubevirtci.

**Comment by mmazur** [comment]
ci test please

**Comment by machacekondra** [comment]
ci test please

**Comment by machacekondra** [comment]
If it will be unstable I would go with k8s for now, until it's resolved.

**Comment by pkliczewski** [comment]
@mmazur can we disable for now when the issues are under investigation? I do not like how our ci is working.

**Comment by pkliczewski** [comment]
@mmazur Are you ok with this change?

File: roles/app_deployment/README.md [hunk 1]
```diff
@@ -13,17 +13,49 @@ Requirements
...
+| egress_allow_all     | UNDEF         | If true allow all egress trafic to virtual machines. |
+| egress_deny_all      | UNDEF         | If true all egress trafic to virtual machines will be denied. |
+| egress_allow_same_namespace | UNDEF  | If true only egress trafic from same namespace will be allowed. |
+
+`ingress`
+| Name                 | Default value |                                            |
+|----------------------|---------------|--------------------------------------------|
+| ingress_ip_blocks    | UNDEF         | This selects particular IP CIDR ranges to allow as ingress sources. These should be cluster-external IPs, since VMI IPs are ephemeral and unpredictable |
+| ingress_namespace_selector | UNDEF   | This selects particular namespaces for which all Pods should be allowed as egress destinations. |
+| ingress_pod_selector | UNDEF         | Specify a vmis to which the ingress rules apply. |
```
Comment (mmazur) [inline roles/app_deployment/README.md]: "specify vmis"

File: roles/app_deployment/README.md [hunk 2]
```diff
@@ -13,17 +13,49 @@ Requirements
...
+| ingress_ports        | UNDEF         | Define ports and protocol of the ports to be allowed by ingress traffic. |
+
+`egress`
+| Name                 | Default value |                                            |
+|----------------------|---------------|--------------------------------------------|
+| egress_ip_blocks     | UNDEF         | This selects particular IP CIDR ranges to allow as ingress sources. These should be cluster-external IPs, since VMI IPs are ephemeral and unpredictable |
+| egress_namespace_selector | UNDEF    | This selects particular namespaces for which all Pods should be allowed as egress destinations. |
+| egress_pod_selector  | UNDEF         | Specify a vmis to which the egress rules apply. |
```
Comment (mmazur) [inline roles/app_deployment/README.md]: ditto

**Pull request merged**